crates/mun_lld/build.rs

```diff
@@ -347,6 +347,9 @@ fn main() {
         println!("cargo:rustc-link-lib=dylib={}", name);
     }
 
+    // xar is used in lld::macho::BitcodeBundleSection
+    println!("cargo:rustc-link-lib=dylib=xar");
```

Comment:
*Problem*:

xar is an optional dep, but i didn't find a way how to check on the fly that lld will be compiled with xar.

Reply:
mm thats whats causing the build failure currently.. Is the version of lld you are using expecting xar?